main.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'codebraker'
+require_relative 'cli'
+
+class Main
+  attr_accessor :current_attempts
```

Comment:
Bad name: it already means  attempts count

Reply:
Fixed

Reply:
Fixed